### PR TITLE
Fix spree_social issues and integrate login with paypal

### PR DESCRIPTION
**Pull request by nitin007**

### CODE REVIEW

File: lib/spree_social/engine.rb [hunk 1]
```diff
@@ -39,8 +40,21 @@ def self.init_provider(provider)
   end
 
   def self.setup_key_for(provider, key, secret)
+    scope = get_scope_for(provider)
+
     Devise.setup do |config|
-      config.omniauth provider, key, secret, setup: true, scope: 'email', info_fields: 'email, name'
+      config.omniauth provider, key, secret, setup: true, scope: scope, sandbox: true, info_fields: 'email, name'
```
Comment (nishant-cyro) [inline lib/spree_social/engine.rb]: @nitin007 - Is sandbox: true required here?

Comment (nitin007) [inline lib/spree_social/engine.rb]: @nishant-cyro actually its required but it should be environment specific. 'sandbox' mode is needed for development environment in paypal auth.

**Comment by nitin007** [comment]
@nishant-cyro Done with the changes

**Comment by nishant-cyro** [comment]
@nitin007 - seems fine :)

**Pull request merged**